README.md

```diff
@@ -8,12 +8,12 @@ This proposal is at stage 0 of [the TC39 process](https://tc39.github.io/process
 
 There are four main combinators in the `Promise` landscape.
 
-| name                 | description                                     |                                                                             |
-| -------------------- | ----------------------------------------------- | --------------------------------------------------------------------------- |
-| `Promise.allSettled` | does not short-circuit                          | [separate proposal](https://github.com/tc39/proposal-promise-allSettled) 🔜 |
-| `Promise.all`        | short-circuits when an input value is rejected  | added in ES2015 ✅                                                          |
-| `Promise.race`       | short-circuits when an input value is settled   | added in ES2015 ✅                                                          |
-| `Promise.any`        | short-circuits when an input value is fulfilled | this proposal 🆕                                                            |
+| name                 | resolve value                              | reject value                               | short-circuits?                  | spec                                                                        |
```

Comment:
typically i would expect this to be called “fulfillment value” and “rejection reason”
```suggestion
| name                 | fulfillment value                          | rejection reason                           | short-circuits?                  | spec                                                                        |
```